scene/2d/camera_2d.cpp

```diff
@@ -568,7 +572,8 @@ Point2 Camera2D::get_camera_screen_center() const {
 
 Size2 Camera2D::_get_camera_screen_size() const {
 	// special case if the camera2D is in the root viewport
-	if (Engine::get_singleton()->is_editor_hint() && get_viewport()->get_parent_viewport() == get_tree()->get_root()) {
+	bool is_editing = get_tree() && get_tree()->get_edited_scene_root() && get_viewport() == get_tree()->get_edited_scene_root()->get_viewport();
```

Comment:
Isn't this equivalent to the method above? Should use that method I'd say and update it